test/rust/intel/src/cpuid.rs

```diff
@@ -7,8 +7,14 @@ pub fn test_cpuid_compile()
 
     let (eax, ebx, ecx, edx) = pal::instruction::execute_cpuid(leaf, subleaf);
 
+    #[cfg(feature = "std")]
```

Comment:
Also if using the suggestion for a `enable_printers` feature, it would be used to enable the tests in each of the files under `test/rust/intel/src/*.rx`. For example, this one would become:

```
#[cfg(feature = "enable_printers")]
```

Reply:
Sure! `feature = "std"` is still used in `src/main.rs` to enable the no_std-specific parts, so things should work when we have a no_std printer implementation (maybe user-supplied symbols).